ada_feeding/config/ada_planning_scene.yaml

```diff
@@ -81,3 +84,10 @@ ada_planning_scene:
       attached: True
       frame_id: forkTip # the frame_id that the pose is relative to
       touch_links: [forkTine] # the links, in addition to frame_id, that should be ignored for collision checking
+    # The offset values for the center coordinates ([x, y, z]) of the table. These 
+    # values are used to translate the detected position into the table's origin.
+    table_detection_offsets: [-0.20, -0.25, -0.79]
+    # A threshold on the distance between the default table quaternion 
+    # and the latest table quaternion to determine whether to reject 
+    # or accept the latest table quaternion
+    quat_dist_thresh: 1.396
```

Comment:
JW, is there some intuitive way to interpret this? Degrees tends to be more intuitive, so if you can say something like `This roughly corresponds to a range of XXX degrees in roll, pitch, and yaw` that would be helpful. I haven't worked much with quaternion differences so am not sure of the conversion, but my hunch is that it can be found with an online search.